src/app/view.component.ts

```diff
@@ -227,14 +227,23 @@ export class ViewComponent implements OnInit {
   public setModel() {
 
   }
+  public StringToArray(str): string {
```

Comment:
This logic doesn't work when loading the GPCP datasets because their depth doesn't go that far in.